src/sysinfo/nvapi_adapter.cpp

```diff
@@ -173,4 +189,20 @@ namespace dxvk {
     bool NvapiAdapter::isVkDeviceExtensionSupported(const std::string name) const { // NOLINT(performance-unnecessary-value-param)
         return m_deviceExtensions.find(name) != m_deviceExtensions.end();
     }
+
+    bool NvapiAdapter::HasNvml() const {
+        return m_nvml.IsAvailable() && m_nvmlDevice != nullptr;
+    }
+
+    std::string NvapiAdapter::NvmlErrorString(nvmlReturn_t result) const {
+        return std::string(m_nvml.ErrorString(result));
+    }
+
+    nvmlReturn_t NvapiAdapter::NvmlDeviceGetTemperature(nvmlTemperatureSensors_t sensorType, unsigned int *temp) const {
+        return m_nvml.DeviceGetTemperature(m_nvmlDevice, sensorType, temp);
+    }
+
+    nvmlReturn_t NvapiAdapter::NvmlDeviceGetUtilizationRates(nvmlUtilization_t *utilization) const {
```

Comment:
`*` should probably bind left to stay consistent with header file. Same in `unsigned int *temp` above.

Reply:
I'm torn about this one, but yeah, let's keep it like the original headers, we are doing the same in all nvapi_* files too. I tried to keep it consistent with c++ style, but often forgot when copying a new method from the headers, so yeah, keeping the original format I guess ensures more consistency.